modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -3740,6 +3794,22 @@ Election timeout expressed in milliseconds.
 
 ---
 
+=== raft_enable_longest_log_detection
+
+Enables an additional step in leader election where a candidate is allowed to wait for all the replies from the node it requested votes from. This may introduce a small delay when recovering from failure but will prevent truncation if any of the replicas have more data than the majority.
```

Comment:
```suggestion
Enables an additional step in leader election where a candidate is allowed to wait for all the replies from the broker it requested votes from. This may introduce a small delay when recovering from failure, but it prevents truncation if any of the replicas have more data than the majority.
```